controllers/authorController.js

```diff
@@ -1,8 +1,48 @@
-const express = require('express'); 
-const router = express.Router(); 
+const express = require('express');
+const router = express.Router();
+const authorService = require('../service/authorService')
 
-router.get('/',function(req,res){
-  res.send("hello");
+router.get('/', function (req, res) {
+  authorService.findAllAuthors()
+    .then((data) => {
+      res.json(data)
+    }).catch(err => {
+      console.errror(err);
+    })
```

Comment:
should send error response to client.
```suggestion
    }).catch(err => {
      res.status(500).json({
        error: err
      });
    });
```